Simplenote/NoticeView.swift

```diff
@@ -0,0 +1,125 @@
+import UIKit
+
+protocol NoticePresentingDelegate: class {
+    func noticePressBegan()
+    func noticePressEnded()
+}
+
+class NoticeView: UIView {
+
+    // MARK: Properties
+    //
+    @IBOutlet private weak var stackView: UIStackView!
+    @IBOutlet private weak var noticeLabel: UILabel!
+    @IBOutlet private weak var noticeButton: UIButton!
+
+    private var handler: (() -> Void)?
+
+    var message: String? {
+        get {
+            noticeLabel.text
+        }
+        set {
+            noticeLabel.text = newValue
+        }
+    }
+
+    var action: NoticeAction? {
```

Comment:
Can we drop this one?

It's coupling the View + Model, which is something you'd generally wanna avoid.

You could, instead, do something like (and maintain the Model <> View details elsewhere)


```
var onAction: (() -> Void)?
var actionTitle: String? {
    get {
        noticeButton.title(for: .normal)
    }
    set {
        noticeButton.setTitle(newValue.title, for: .normal)
    }
}
```

Reply:
Yup, like that much better